src/components/Tool/FixMailingAddresses/FixMailingAddresses.tsx

```diff
@@ -282,7 +282,10 @@ const FixMailingAddresses: React.FC<Props> = ({
                 addresses: {
                   nodes: [
                     ...contact.addresses.nodes,
-                    createdAddressData.createAddress.address,
+                    {
+                      ...createdAddressData.createAddress.address,
```

Comment:
Same thing here about adding `startDate` to the mutation so that we can remove these lines.